digos-ambassador/Content/Transformations/Species/snake/Body.yml

```diff
@@ -0,0 +1,9 @@
+part: Body
+species: snake
+description: "Transforms the target's body into that of an anthro snake."
+default_base_colour: "green"
+is_nsfw: false
+shift_message: "{@target}'s body shifts, flesh undulating as it becomes lean and powerful beneath a growing tidal wave of smooth {@colour} scales. Their spine slightly lengthens to give their now flexible torso an exotic and striking appearance, and the scales on their belly turn a lighter hue of {@colour} to create a distinct two-toned look."
+grow_message: "{@target} stretches and grows outward, forming a long muscular body covered in smooth {@colour} scales. Their long spine gives their torso an exotic and striking appearance,and the scales on their belly hold a lighter hue of {@colour} to create a distinct two-toned look."
```

Comment:
Missing space after "striking appearance," between the comma and following word.

Reply:
Second sentence of the grow message is also in perfect present form instead of progressive present.